CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ This file documents all changes to Argus-frontend. This file is primarily meant
 ### Fixed
 - Fix ARGUS logo clipping part of the name.
 
+- End time to the incident detailed view (for closed incidents only)
+
 ### Changed
 - Logo and favicon
 - Add seconds to timestamps in elements of the event feed in detailed incident view
+
+### Added
+- End time to the incident detailed view (for closed incidents only)
```

Comment:
This should probably only be under the `ADDED` section.

Reply:
But it is already only under `ADDED` section. 😅

```
### Changed
- Add seconds to timestamps in elements of the event feed in detailed incident view
```
is there from a different PR, if you meant to remove that one